src/pages/iou/request/step/IOURequestStepDescription.js

```diff
@@ -0,0 +1,122 @@
+import {useFocusEffect} from '@react-navigation/native';
+import lodashGet from 'lodash/get';
+import React, {useCallback, useRef} from 'react';
+import {View} from 'react-native';
+import FormProvider from '@components/Form/FormProvider';
+import InputWrapperWithRef from '@components/Form/InputWrapper';
+import TextInput from '@components/TextInput';
+import transactionPropTypes from '@components/transactionPropTypes';
+import useLocalize from '@hooks/useLocalize';
+import * as Browser from '@libs/Browser';
+import compose from '@libs/compose';
+import Navigation from '@libs/Navigation/Navigation';
+import updateMultilineInputRange from '@libs/UpdateMultilineInputRange';
```

Comment:
```suggestion
import updateMultilineInputRange from '@libs/updateMultilineInputRange';
```
App crashes on main right now